rs/config/src/execution_environment.rs

```diff
@@ -52,6 +52,12 @@ const SUBNET_WASM_CUSTOM_SECTIONS_MEMORY_CAPACITY: NumBytes = NumBytes::new(2 *
 /// The number of bytes reserved for response callback executions.
 const SUBNET_MEMORY_RESERVATION: NumBytes = NumBytes::new(10 * GIB);
 
+/// The soft limit on the subnet-wide number of callbacks. Beyond this
+pub const SUBNET_CALLBACK_SOFT_CAP: usize = 1_000_000;
+
+/// The number of callbacks that are guaranteed to each canister.
+pub const CANISTER_CALLBACK_QUOTA: usize = 50;
```

Comment:
I've renamed them to "soft limit" (since the "soft" part is important; this is different from e.g. the message memory limit, which is a "hard" limit); and, respectively, "guaranteed callback quota" (since this is the namie we've been using in our design docs and discussions for almost a year now).